### ISSUE TITLE
Realm.build/module.modulemap:2:21: Umbrella header 'Realm.h' not found

### ISSUE THREAD

**Issue by intrigua** [realm/realm-cocoa#3660]
## Goals

I'm trying to update CocoaPods to 1.0.0, but cannot build RealmSwift

## Actual Results

Get this error: Umbrella header 'Realm.h' not found

## Steps to Reproduce

I've made a test project here: https://github.com/intrigua/RealmTest
It's just a template project with one pod in the podfile.
I did `pod install` and tried to build but failed with this error.
Removing all of these folders: ./Pods, ~/Library/Caches/CocoaPods, Xcode DerivedData doesn't help.

## Version of Realm and Tooling

```
ProductName:	Mac OS X
ProductVersion:	10.11.5
BuildVersion:	15F34

/Applications/Xcode.app/Contents/Developer
Xcode 7.3.1
Build version 7D1014

~/.gem/ruby/2.0.0/bin/pod
1.0.0
Realm (1.0.0)
RealmSwift (1.0.0)

/bin/bash
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)


(not in use here)

/usr/bin/git
git version 2.7.4 (Apple Git-66)
```

**Comment by jpsim** [comment]
Are you using Realm in an OS X app? If so, this is probably https://github.com/CocoaPods/CocoaPods/issues/5313 which was fixed in https://github.com/CocoaPods/CocoaPods/issues/5352 which will be included in the next release of CocoaPods.

**Comment by intrigua** [comment]
Yes, it's OS X app

**Comment by jpsim** [comment]
Alright, I apologize for the inconvenience then, it's almost certainly https://github.com/CocoaPods/CocoaPods/issues/5313. Until CocoaPods releases a version including the fix, I encourage you to use CocoaPods 0.39.0 or CocoaPods built from `master`. We're also tracking all this in #3570.
